components/HTMLContent.js

```diff
@@ -120,6 +120,11 @@ const HTMLContent = styled(({ content, collapsable, sanitize, ...props }) => {
         height: 100%;
       }
     }
+    &[data-trix-content-type='--embed-iframe-anchorFm'] {
+      iframe {
+        min-height: 300px;
```

Comment:
AnchorFm has a min-height set on its content. This prevents adding a scrollbar on the iframe.